suzieq/engines/pandas/address.py

```diff
@@ -33,17 +33,18 @@ def get(self, **kwargs) -> pd.DataFrame:
 
         addr = kwargs.pop("address", [])
         prefix = kwargs.pop("prefix", [])
-        columns = kwargs.get("columns", [])
+        columns = kwargs.pop("columns", [])
         ipvers = kwargs.pop("ipvers", "")
         user_query = kwargs.pop("query_str", "")
+        addnl_fields = kwargs.pop('addnl_fields', [])
```

Comment:
Here we lose 'master' as default addnl_field. Is it calculated or a mistake?

Reply:
Nope, master is an alias for vrf. If you notice, the address schema has no master.